src/Bundle/FrameworkBundle/fixtures/BarCommand.js

```diff
@@ -0,0 +1,21 @@
+const Command = Jymfony.Component.Console.Command.Command;
+
+class BarCommand extends Command {
+    configure() {
+        this.name = 'command:bar';
+        this.description = 'The command:bar command';
+    }
+
+    * interact(input, output) {
```

Comment:
Not a generator.
While overriding these methods you're not forced to define them as generators. Give the Async runner an hint telling it that you're not `yield`-ing any promise.